### ISSUE TITLE
UI : proposition to inverse tab tools and parcel tools

### ISSUE THREAD

**Issue by MaelREBOUX** [georchestra/mapstore2-cadastrapp#92]
Here the current UI 
![image](https://user-images.githubusercontent.com/6370443/109021325-13ea0d80-76bb-11eb-825e-fb08930c350f.png)

As the list of the parcels is IN a tab, for me the order is tab1 > list1 | tab2 > list2

As I want to interact with the parcel, for example to delete on of the current list, I instinctively search the very near tools. Because this tools are used very often.

But the first tool when the mouse go to the top is the tools for the tabs. Not logical for me.

So I propose to inverse this 2 groups of buttons.

**Comment by tdipisa** [comment]
@MaelREBOUX this is the original design we proposed. For this reason, we need to consider it as an enhancement and put it in blocked for the moment to concentrate the work on eventual bugs you find during tests.

**Comment by MaelREBOUX** [comment]
No pb : it appeared as working a long with it.

And it has to be debate with @jusabatier

**Comment by landryb** [comment]
i have to agree with @MaelREBOUX , the placement of buttons groups doesnt feel natural as is.

Maybe it would be simpler for tab handling buttons to be like browser tabs, ie have a cross on the tab itself to close it, and a plus button next to the last tab to create one ? this way, there's only a single button group left.

And since below the parcel list, there's an empty space (on the right there's a label with `0 Objet  (0 Sélectionné)`), maybe it would make sense to have the buttons working on the content of the list just below the list, on the left, aligned with the label ? this way they're the closest possible to the parcel checkbox list.
